Expo1900/Expo1900/Controller/ExpoItemListViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ExpoItemListViewController.swift
+//  Expo1900
+//
+//  Created by 세홍, 송준, 무리 on 2023/02/22.
+//
+
+import UIKit
+
+final class ExpoItemListViewController: UIViewController {
+    
+    @IBOutlet weak var listTableView: UITableView!
+    
+    var expoItems: [ExpoItem] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        listTableView.dataSource = self
+        decodeData()
+        self.navigationController?
+            .navigationBar
+            .topItem?
+            .backBarButtonItem = UIBarButtonItem(title: "메인",
+                                                 style: .plain,
+                                                 target: self,
+                                                 action: nil)
+    }
+    
+    private func decodeData() {
+        let jsonDecoder: JSONDecoder = JSONDecoder()
+        guard let dataAsset: NSDataAsset = NSDataAsset(name: "items") else { return }
+        
+        do {
+            self.expoItems = try jsonDecoder.decode([ExpoItem].self, from: dataAsset.data)
+        } catch {
+            print(error.localizedDescription)
+        }
+        self.listTableView.reloadData()
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard let nextViewController: ItemDetailViewController = segue.destination as? ItemDetailViewController else { return }
+        
+        guard let cell: ListTableViewCell = sender as? ListTableViewCell else { return }
```

Comment:
굳이 guard문을 두개로 나눈 이유가 있을까요?
```suggestion
        guard let nextViewController: ItemDetailViewController = segue.destination as? ItemDetailViewController,
                   let cell: ListTableViewCell = sender as? ListTableViewCell else { 
                   return 
        }
```

Reply:
- `segue`에 대한 부분을 먼저 선언해놓은 후에 `cell`을 선언하여 따로 쓰게된것같습니다! 코멘트 참고하여 수정하였습니다!

e0754babeaa98c5a392575db2e11412d22d69129